hello/lib/hello_web/templates/page/index.text.eex

```diff
@@ -0,0 +1 @@
+This a good text to see, happy face :)
```

Comment:
👀

Reply:
I already fixed it, added a new line at the end of the file and changed the text.